src/symbols.cs

```diff
@@ -0,0 +1,215 @@
+
+		[Export ("effectWithCumulative")]
+		NSSymbolVariableColorEffect Cumulative { get; }
+
+		[Export ("effectWithReversing")]
+		NSSymbolVariableColorEffect Reversing { get; }
+
+		[Export ("effectWithNonReversing")]
+		NSSymbolVariableColorEffect NonReversing { get; }
+
+		[Export ("effectWithHideInactiveLayers")]
+		NSSymbolVariableColorEffect HideInactiveLayers { get; }
+
+		[Export ("effectWithDimInactiveLayers")]
+		NSSymbolVariableColorEffect DimInactiveLayers { get; }
+	}
+
+	[Watch (10, 0), TV (17, 0), Mac (14, 0), iOS (17, 0), MacCatalyst (17, 0)]
+	[BaseType (typeof (NSSymbolEffect))]
+	interface NSSymbolScaleEffect {
+		[Static]
+		[Export ("effect")]
+		NSSymbolScaleEffect Create ();
+
+		[Static]
+		[Export ("scaleUpEffect")]
+		NSSymbolScaleEffect CreateScaleUpEffect ();
+
+		[Static]
+		[Export ("scaleDownEffect")]
+		NSSymbolScaleEffect CreateScaleDownEffect ();
+
+		[Export ("effectWithByLayer")]
+		NSSymbolScaleEffect ByLager { get; }
+
+		[Export ("effectWithWholeSymbol")]
+		NSSymbolScaleEffect WholeSymbol { get; }
+	}
+
+	[Watch (10, 0), TV (17, 0), Mac (14, 0), iOS (17, 0), MacCatalyst (17, 0)]
+	[BaseType (typeof (NSSymbolEffect))]
+	interface NSSymbolAppearEffect {
+		[Static]
+		[Export ("effect")]
+		NSSymbolAppearEffect Create ();
+
+		[Static]
+		[Export ("appearUpEffect")]
+		NSSymbolAppearEffect CreateAppearUpEffect ();
+
+		[Static]
+		[Export ("appearDownEffect")]
+		NSSymbolAppearEffect CreateAppearDownEffect ();
+
+		[Export ("effectWithByLayer")]
+		NSSymbolAppearEffect ByLager { get; }
+
+		[Export ("effectWithWholeSymbol")]
+		NSSymbolAppearEffect WholeSymbol { get; }
+	}
+
+	[Watch (10, 0), TV (17, 0), Mac (14, 0), iOS (17, 0), MacCatalyst (17, 0)]
+	[BaseType (typeof (NSSymbolEffect))]
+	interface NSSymbolDisappearEffect {
+		[Static]
+		[Export ("effect")]
+		NSSymbolDisappearEffect Create ();
+
+		[Static]
+		[Export ("disappearUpEffect")]
+		NSSymbolDisappearEffect CreateDisappearUpEffect ();
+
+		[Static]
+		[Export ("disappearDownEffect")]
+		NSSymbolDisappearEffect CreateDisappearDownEffect ();
+
+		[Export ("effectWithByLayer")]
+		NSSymbolDisappearEffect ByLager { get; }
+
+		[Export ("effectWithWholeSymbol")]
+		NSSymbolDisappearEffect WholeSymbol { get; }
+	}
+
+	[Watch (10, 0), TV (17, 0), Mac (14, 0), iOS (17, 0), MacCatalyst (17, 0)]
+	[BaseType (typeof (NSSymbolContentTransition))]
+	interface NSSymbolReplaceContentTransition {
+		[Static]
+		[Export ("transition")]
+		NSSymbolReplaceContentTransition CreateTransition ();
+
+		[Static]
+		[Export ("replaceDownUpTransition")]
+		NSSymbolReplaceContentTransition CreateReplaceDownUpTransition ();
+
+		[Static]
+		[Export ("replaceUpUpTransition")]
+		NSSymbolReplaceContentTransition CreateReplaceUpUpTransition ();
+
+		[Static]
+		[Export ("replaceOffUpTransition")]
+		NSSymbolReplaceContentTransition CreateReplaceOffUpTransition ();
+
+		[Export ("transitionWithByLayer")]
+		NSSymbolReplaceContentTransition ByLager { get; }
```

Comment:
```suggestion
		NSSymbolReplaceContentTransition ByLayer { get; }
```

🍻

Reply:
Guess who got a typo in the search and replace 😅